docs/en/esptool/advanced-options.rst

```diff
@@ -117,3 +117,18 @@ An example of this is available in the :ref:`merge_bin <merge-bin>` command desc
 .. note:: PowerShell users
 
     Because of `splatting <https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_splatting?view=powershell-7.3>`__ in PowerShell (method of passing a collection of parameter values to a command as a unit) there is a need to add quotes around @filename.txt ("@filename.txt") to be correctly resolved.
+
+Filtering serial ports
+----------------------
+.. _filtering_serial_ports:
+
+``--port-filter <FilterType>=<FilterValue>`` allows limiting ports that will be tried. This can be useful when esptool is run on a system
+with many serial ports. There are a few different types that can be combined. A port must match all specified FilterTypes, and must match
+at least one FilterValue for each specified FilterType to be considered. Example filter configurations:
+
+.. list::
+
+    * ``--port-filter vid=0x303A`` matches ports with the Espressif USB VID.
+    * ``--port-filter vid=0x303A --port-filter vid=0x0403`` matches Espressif and FTDI ports by VID.
+    * ``--port-filter vid=0x303A --port-filter pid=0x1001`` matches Espressif ESP32-S3 ports by VID and PID.
```

Comment:
`303A`:`1001` is not actually just the  ESP32-S3, but the USB-Serial/JTAG unit in general. That means any chip with this peripheral will show up with this VID:PID (S3, C3, C6, H2, P4, ...). 

If you want to mention a specific chip, USB-OTG mode has to be used. e.g. `ESP32-S2` = `VID:PID=303A:0002`